packages/protobuf-test/typescript/tsconfig.4_1_2.json

```diff
@@ -1,6 +1,16 @@
 {
-  "include": ["../src/gen/js/extra/*.js", "../src/gen/ts/extra/*.ts"],
-  // These are the default compiler options for TypeScript v4.1.2, created with `tsc --init`
+  "include": ["../src/**/*"],
+  "exclude": [
+    // Exclude WKTs and their usage in tests because they use syntax not
+    // compatible with older versions of TypeScript.  WKTs used by actual
+    // consumers of this library are generated with ts-nocheck so its safe
+    // to ignore them here and still get version compatibility assurance.
+    "../src/gen/ts/google/protobuf/*.ts",
+    "../src/google/protobuf/*.test.ts",
+    "../src/message.test.ts"
```

Comment:
We should be able to include message.test.ts, if we change the test to use `Timestamp` from `@bufbuild/protobuf` instead of the locally generated file.